runtime/calamari/src/lib.rs

```diff
@@ -333,6 +333,7 @@ impl pallet_multisig::Config for Runtime {
 impl pallet_utility::Config for Runtime {
 	type Event = Event;
 	type Call = Call;
+	type PalletsOrigin = OriginCaller;
```

Comment:
This basically allows Root to become any other origin type like collectives.
Related https://github.com/paritytech/substrate/pull/9934